.README/rules/tag-lines.md

```diff
@@ -0,0 +1,28 @@
+### `tag-lines`
+
+Enforces lines (or no lines) between tags.
+
+#### Options
+
+The first option is a single string set to "always" or "never" (defaults to
+"never").
+
+The second option is an object with the following optional properties.
+
+##### `count` (defaults to 1)
+
+Use with "always" to indicate the number of lines to require be present.
+
+##### `noEndLine` (defaults to `false`)
+
+Use with "always" to indicate tag lines should not be added at the end.
```

Comment:
I don't understand this option. What is a tag line?

Reply:
Clarified to state:

> Use with "always" to indicate the normal lines after tags should not be
added after the final tag.

Also changed `noEndLine` to `noEndLines` as can be more than one between tags normally.